assets/pages/oneoff-contributions/reducers/reducers.js

```diff
@@ -39,6 +40,11 @@ export type CombinedState = {
   csrf: CsrfState,
 };
 
+export type PageState = {
```

Comment:
Is it possible to avoid declaring so many types with the same name? I would name this one `OneOffContributionsPageState`. It is not weird here but when you import it somewhere else you have to read the entire import to check what you are importing.

Reply:
I did think about that, but then the name becomes very verbose for what is essentially a tiny wrapper class. Also in a sense all the different types are really the same thing - a wrapper around the page object so calling them the same thing makes that explicit and then you can differentiate between them according to where they sit in the codebase. In short I would vote to leave them as is, but happy to discuss further if you like.

Reply:
I understand why you did it, my only concern is if a new developer reads the code base and would be more difficult to understand why these different types have the same name. I prefer to be explicit about the differences of the type, but again is my personal point of view.